lib/cocoapods/bazel/config.rb

```diff
@@ -53,7 +54,8 @@ class Config
         buildifier: true,
         default_xcconfigs: {}.freeze,
         features: {
-          experimental_deps_debug_and_release: false
+          experimental_deps_debug_and_release: false,
+          build_file_doc: ""
```

Comment:
Do we need to make it a feature? It's not a breaking change, right?

Reply:
it's a draft :) going to make it a proper PR soon

Reply:
It'll default to an empty string if not supplied, won't it? In other words not a breaking change?

Reply:
yup it's not a experimental feature, i moved it out now 🙏